memcached/internal/alloc.c

```diff
@@ -1,2 +1,38 @@
 #include <tarantool/module.h>
 #include <small/slab_cache.h>
+#include <small/quota.h>
+
+struct slab_arena arena;
+struct slab_cache slabc;
+struct quota quota;
+
+void
+memcached_slab_arena_create()
+{
+	static __thread bool inited = false;
+	if (inited) return;
+	size_t quota_size = UINT_MAX;
```

Comment:
Why don't use `QUOTA_MAX`? We do that in tarantool for the runtime arena.

Reply:
Fixed.

```diff
--- a/memcached/internal/alloc.c
+++ b/memcached/internal/alloc.c
@@ -11,7 +11,7 @@ memcached_slab_arena_create()
 {
        static __thread bool inited = false;
        if (inited) return;
-       size_t quota_size = UINT_MAX;
+       size_t quota_size = QUOTA_MAX;
        quota_init(&quota, quota_size);
        /* Parameters are the same as in src/lib/core/fiber.c */
        const size_t SLAB_SIZE = 4 * 1024 * 1024;

```